.gitattributes

```diff
@@ -1,2 +1,4 @@
 *.java text=auto
 *.xml text=auto
+text eol=lf
+
```

Comment:
This seems wrong for Windows users who use crlf with text file line endings.

```suggestion
```

Removing the changes to this file will also make your PR build a lot faster. :wink: